extensions/aws2-sns/runtime/pom.xml

```diff
@@ -31,7 +33,7 @@
     <description>The aws2-sns component is used for sending messages to an Amazon Simple Notification Topic.</description>
 
     <properties>
-        <firstVersion>1.0.0-M6</firstVersion>
+        <firstVersion>1.0.0-M7</firstVersion>
```

Comment:
I think keeping it M6 would be correct - that's the real first version although only JVM was supported there.

Reply:
fixed